### PR TITLE
IsIPinRanges - better handling of ip ranges array

### PR DESCRIPTION
**Pull request by GavrielFilippov**
customer had a problem where a spaces or new lines in the provided list caused the command to return wrong output

### CODE REVIEW

File: Scripts/script-IsIPInRanges.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ script: |
   }
 
   var isInRange = ipranges.some(function(iprange) {
-      return isIPInSubnet(ip, iprange);
+      return isIPInSubnet(ip, iprange.replace(/\n/g,'').replace(/\r/g,'').replace(/ /g,''));
```
Comment (aviadl) [inline Scripts/script-IsIPInRanges.yml]: This will work, but why not create one regex for this [\n\g ] ?

**Comment by Itay4** [comment]
@aviadl Can you please review?

File: Scripts/script-IsIPInRanges.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ tags:
 - ip
 - Condition
 comment: Returns yes if ip is in one of the ranges provided, returns no otherwise.
-system: true
+enabled: true
```
Comment (aviadl) [inline Scripts/script-IsIPInRanges.yml]: change this line back to system:true

File: Scripts/script-IsIPInRanges.yml [hunk 3]
```diff
@@ -60,3 +60,5 @@ outputs:
 - contextPath: IP.InRange
   description: Is the IP is in the input ranges? (could be 'yes' or 'no)
 scripttarget: 0
+runonce: false
```
Comment (aviadl) [inline Scripts/script-IsIPInRanges.yml]: remove this line

File: Scripts/script-IsIPInRanges.yml [hunk 4]
```diff
@@ -60,3 +60,5 @@ outputs:
...
+releaseNotes: "Better handling of IP ranges array"
```
Comment (aviadl) [inline Scripts/script-IsIPInRanges.yml]: I think that this release note is incorrect
At lease this is not what your fix is doing

**Comment by Itay4** [comment]
@GavrielFilippov What is the status of this?

**Pull request closed without merging**